tests/nvapi_d3d12.cpp

```diff
@@ -5,6 +5,19 @@
 
 using namespace trompeloeil;
 
+typedef struct _NVAPI_D3D12_RAYTRACING_GEOMETRY_DESC_EX_V1 {
+    NVAPI_D3D12_RAYTRACING_GEOMETRY_TYPE_EX type; //!< The type of geometry stored in the union of this structure.
+    D3D12_RAYTRACING_GEOMETRY_FLAGS flags;        //!< Flags affecting how this geometry is processed by the raytracing pipeline.
+    union {
+        D3D12_RAYTRACING_GEOMETRY_TRIANGLES_DESC triangles;              //!< Describes triangle geometry if \c type is #NVAPI_D3D12_RAYTRACING_GEOMETRY_TYPE_TRIANGLES_EX.
+                                                                         //!< Otherwise, this parameter is unused (space repurposed in a union).
+        D3D12_RAYTRACING_GEOMETRY_AABBS_DESC aabbs;                      //!< Describes AABB geometry if \c type is #NVAPI_D3D12_RAYTRACING_GEOMETRY_TYPE_PROCEDURAL_PRIMITIVE_AABBS_EX.
+                                                                         //!< Otherwise, this parameter is unused (space repurposed in a union).
+        NVAPI_D3D12_RAYTRACING_GEOMETRY_OMM_TRIANGLES_DESC ommTriangles; //!< Describes triangle geometry which may optionally use Opacity Micromaps, if \c type is #NVAPI_D3D12_RAYTRACING_GEOMETRY_TYPE_OMM_TRIANGLES_EX.
+                                                                         //!< Otherwise, this parameter is unused (space repurposed in a union).
+    };
+} NVAPI_D3D12_RAYTRACING_GEOMETRY_DESC_EX_V1;
```

Comment:
What about calling this `NVAPI_D3D12_RAYTRACING_GEOMETRY_DESC_EX_R530` and adding a comment that DMM has been added in 535 without a new struct version. 
Calling this `V1` is imho confusing since  the "usual" struct versioning isn't applied here.

Reply:
I'll check first if opacity micromaps weren't actually introduced earlier than in 530 release, if so then I'll name it after the first release it was introduced in.

By the way, is this a good place for such ad-hoc `typedef struct`? I was somewhat conflicted about where I should put it.

Reply:
Okay, then for now I'll keep it here and when (if) we have more structures needed only in tests then we could reconsider moving them to a separate header.